Xamarin.Forms.Core.Design/AttributeTableBuilder.cs

```diff
@@ -1,8 +1,7 @@
 ﻿using System;
 using System.ComponentModel;
 using System.Linq;
-using System.Windows.Markup;
-using Microsoft.Windows.Design;
+using Microsoft.VisualStudio.DesignTools.Extensibility;
```

Comment:
I would suggest only using the Microsoft.VisualStudio.DesignTools.Extensibility namespace for XmlnsSupportsValidationAttribute to be safe and not replacing the entire `using Microsoft.Windows.Design;` with it.

There may be other attributes where it might matter.